og.module

```diff
@@ -152,6 +154,25 @@ function og_entity_field_access($operation, FieldDefinitionInterface $field_defi
 }
 
 /**
+ * Implements hook_entity_bundle_field_info().
+ */
+function og_entity_bundle_field_info(EntityTypeInterface $entity_type, $bundle, array $base_field_definitions) {
+  // Add a read only property to group entities as a group flag.
+  if (Og::isGroup($entity_type->id(), $bundle)) {
```

Comment:
It's a matter of taste, but I prefer early returns, to save some indentation, so:

```php
if (Og::isGroup($entity_type->id(), $bundle)) {
  return [];
}
...
```